.github/workflows/build.yml

```diff
@@ -9,8 +9,10 @@ jobs:
       - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
         with:
           submodules: true
+      - name: install python3-requests
+        run: pip3 install requests
```

Comment:
Could you pin the hash and version?

Reply:
Same as https://github.com/NordSecurity/libtelio/pull/543#discussion_r1624164533